querydsl-sql/src/test/java/com/querydsl/sql/types/TypeTest.java

```diff
@@ -45,7 +45,29 @@
 
     @Override
     public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
-        if (method.getName().startsWith("get")) {
+        if (method.getName().equals("wasNull")) {
```

Comment:
It doesn't matter here since it's just a test, but I like to call `equals` on the constant string (`"wasNull".equals(method.getName());`) to eliminate any chance of an NPE.